jwala-webapp/src/main/java/com/cerner/jwala/web/javascript/variable/property/ApplicationPropertySourceDefinition.java

```diff
@@ -18,17 +18,20 @@
     JWALA_ROLE_ADMIN("jwalaRoleAdmin", "jwala.role.admin", "Tomcat Admin", VariableStyle.STRING),
     TOMCAT_MANAGER_PROTOCOL("tomcatManagerProtocol", "tomcat.manager.protocol", "http", VariableStyle.STRING),
     APACHE_HTTPD_STATUS_PING_RESOURCE("apacheHttpdStatusPingResource", "apache.httpd.status.ping.resource", "index.html", VariableStyle.STRING),
-    TOMCAT_IMAGE_LOGO_PATH("tomcatImageLogoPath", "tomcat.image.logo.path", "/tomcat-power.gif", VariableStyle.STRING);
+    TOMCAT_IMAGE_LOGO_PATH("tomcatImageLogoPath", "tomcat.image.logo.path", "/tomcat-power.gif", VariableStyle.STRING),
+    JWALA_CLIENT_NAME("jwalaClientName", "jwala.client.name", "Cerner Corporation", VariableStyle.STRING),
+    JWALA_HHRR("jwalaHhrr","jwala.hhrr", "B007", VariableStyle.STRING),
```

Comment:
Could the default value be blank instead of B007 😆

Reply:
+1

Reply:
Actually after thinking about @Sharvaribarve's comment, do we need default values for client name as well ? Maybe we can ask the team.

Reply:
Yes, I agree to making all fields default to blank. I will ask team before doing it.